src/components/Button/Button.jsx

```diff
@@ -0,0 +1,43 @@
+import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
+const ACTION_FIELD_REVERSE = 'Reverse';
+const ACTION_FIELD_RESET = 'Reset';
+
+export const Button = ({ btn, searchField,
+  setSearchField, goodsRender, isReversed, setIsReversed,
+  setGoodsRender }) => {
+  const setFieldFunc = () => {
+    switch (btn) {
+      case ACTION_FIELD_REVERSE:
+        setIsReversed(prev => !prev);
+        break;
+      case ACTION_FIELD_RESET:
+        setIsReversed(false);
+        setSearchField('');
+        break;
+      default:
+        setIsReversed(false);
+        setSearchField({ btn });
```

Comment:
why do you need to send the object here

Reply:
to make useEffect work every time when button is pressed